bigquery/indexer.py

```diff
@@ -71,6 +75,24 @@ def convert_to_index_name(s):
 
 def init_elasticsearch(elasticsearch_url, index_name):
     es = Elasticsearch([elasticsearch_url])
+
```

Comment:
nit: ideally, since this code block is necessary only in dev using docker-compose, I'd prefer to extract this out to something like > docker-compose calls main.py > main.py runs this loop waiting for es to come up then calls indexer.py > indexer.py remains largely unchanged

Reply:
This code is necessary for any server that talks to Elasticsearch. (That's why I filed [an issue](https://github.com/elastic/elasticsearch-py/issues/778) to provide a helper in Python client library.)

Anytime you start API server, you need to wait for Elasticsearch to come up. It just so happens that I manually deployed ES on GKE, then a while later ran indexer on GKE. You could imagine that for CD (continuous deployment), ES and indexer would start at the same time on GKE and this code is needed.